src/lib.rs

```diff
@@ -687,6 +689,12 @@ impl MetadataCommand {
         self
     }
 
+    /// Set whether to inherit stderr from parent
+    pub fn inherit_stderr(&mut self, inherit: bool) -> &mut MetadataCommand {
```

Comment:
That's what happens underneath, but I think the API should be more about how it appears for users. So something about printing stderr? or showing errors?

Do we even need this? We could alternatively return stderr from `exec` so it can be accessed as a string

Reply:
As the issue reported, cargo could be blocked for a long time, capture stderr then print does not work because it will be only be printed after the file lock is released.

Reply:
ah right, that was the use case, forgot again. thanks.

So only my first point of naming this better stands.

Reply:
Originally I used `show_output` but it doesn't say what it will output. Do you have any suggestions in mind?

Reply:
Maybe just `verbose`? Idk, I don't have a good idea either. Even `show_stderr` seems better than `inherit_stderr` tho

Reply:
`verbose` sounds good.